tests/test_model.py

```diff
@@ -14,6 +15,11 @@
 from flask_sqlalchemy.model import Model
 
 
+class UTCNow(datetime):
```

Comment:
Rather than making this a subclass with a tricky `__new__`, this should just be a function:

```python
def now() -> datetime:
    return datetime.now(timezone.utc)
```